nbs/23_tutorial.vision.ipynb

```diff
@@ -495,7 +494,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This time we resized to a larger size before batching, and we add `batch_tfms`. `aug_transforms` is a function that provides a collection of data augmentation transforms with defaults we found worked very well on most datasets (you can customize each one by passing the right arguments)."
+    "This time we resized to a larger size before batching, and we add `batch_tfms`. `aug_transforms` is a function that provides a collection of data augmentation transforms with defaults that are observed to perform well on most datasets.`aug_trsnforms` can be cutomized to suit one's need."
```

Comment:
```suggestion
    "This time we resized to a larger size before batching, and we added `batch_tfms`. `aug_transforms` is a function that provides a collection of data augmentation transforms with defaults we found that perform well on many datasets.  You can customize these transforms by passing appropriate arguments to `aug_transnforms`."
```

Reply:
Don't think you should change the original meaning of this sentence?  I'll make some small modifications to revert parts of it back